bin/build-metadata.js

```diff
@@ -134,11 +133,13 @@ const aliases = {
 
 function getLibPhoneNumberData() {
 	return new Promise( function ( resolve, reject ) {
-		request.get( LIBPHONENUMBER_METADATA_URL, function ( error, response, body ) {
-			if ( error || response.statusCode >= 400 ) {
-				throw error || response.statusCode;
+		fetch( LIBPHONENUMBER_METADATA_URL ).then( async ( response ) => {
+			if ( response.status >= 400 ) {
+				throw response.status;
```

Comment:
This code now becomes a strange mix of chained promises and async functions. It's no longer necessary to wrap the whole block in `new Promise()` because we're no longer dealing with callbacks.

Let's rewrite `getLibPhoneNumberData` to an async function. Remove two levels of indentation, remove the `try`/`catch` block around `script.runInNewContext` (it merely re-throws the error anyway).